elixir/apps/api/lib/api/gateway/views/client.ex

```diff
@@ -7,7 +7,12 @@ defmodule API.Gateway.Views.Client do
       public_key: client.public_key,
       preshared_key: preshared_key,
       ipv4: client.ipv4,
-      ipv6: client.ipv6
+      ipv6: client.ipv6,
+      version: client.last_seen_version,
+      device_serial: client.device_serial,
+      firebase_installation_id: client.firebase_installation_id,
+      identifier_for_vendor: client.identifier_for_vendor,
+      device_uuid: client.device_uuid
```

Comment:
Sorry - one more thing I just remembered admins will certainly want here is device OS / client type:



```suggestion
      device_uuid: client.device_uuid,
      user_agent: client.last_seen_user_agent,
      client_type: if subject.actor.type == :service_account, do: "Headless", else: "GUI"
```

Reply:
The user agent already contains the version, should we remove that one then?